recipes/json-schema-validator/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup()
+
+find_package(nlohmann_json_schema_validator REQUIRED)
+
+add_executable(example example.cpp)
+set_property(TARGET example PROPERTY CXX_STANDARD 11)
+
+target_link_libraries(example nlohmann_json_schema_validator::nlohmann_json_schema_validator)
```

```suggestion
target_link_libraries(example ${CONAN_LIBS})
```

Keep cmake 2.8 features